dotnetv3/AutoScaling/Tests/AutoScalingWrapperTests.cs

```diff
@@ -0,0 +1,321 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+using Amazon.AutoScaling;
+using Amazon.AutoScaling.Model;
+using Amazon.EC2;
+using AutoScalingActions;
+
+namespace AutoScalingTests
+{
+    public class AutoScalingWrapperTests
+    {
+        private readonly IConfiguration _configuration;
+        private readonly AmazonAutoScalingClient _autoScalingClient;
+        private readonly AutoScalingWrapper _autoScalingWrapper;
+        private readonly AmazonEC2Client _ec2Client;
+        private readonly EC2Wrapper _ec2Wrapper;
+        private readonly AmazonCloudWatchClient _cloudWatchClient;
+        private readonly CloudWatchWrapper _cloudWatchWrapper;
+
+        // the Amazon Resource Name (ARN) of the IAM service linked role.
```

Comment:
```suggestion
        // The Amazon Resource Name (ARN) of the AWS Identity and Access Management (IAM) service-linked role.
```

Reply:
The comment seems to reference the ARN , but I don't see it here, I see the group name. Is the comment on the wrong line?